{
  "id": "MAL-2025-2937",
  "modified": "2025-03-28T13:05:19Z",
  "published": "2025-03-28T13:05:19Z",
  "summary": "Malicious code in asyncaiosignal (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "asyncaiosignal",
        "purl": "pkg:pypi/asyncaiosignal"
      },
      "versions": [
        "0.3",
        "0.4",
        "0.5"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/asyncaiosignal/MAL-2025-2937.json"
      }
    }
  ]
}